src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/BooleanExpressionComplexityCheck.java

```diff
@@ -30,8 +30,8 @@
 
 /**
  * <p>
- * Restrict the number of {@code &amp;&amp;}, {@code ||}, {@code &amp;}, {@code |} and
- * {@code ^} in an expression.
```

Comment:
Should we keep "in an expression" because or the name of the check. Also `|` can not be in a boolean expression like a catch. Ex: `catch(Exception | Exception)`

Reply:
added `in an expression`